core/templates/dev/head/filters.js

```diff
@@ -188,7 +188,11 @@ oppia.filter('parameterizeRuleDescription', [
             replacementText = '[';
             var key = inputs[varName];
             for (var i = 0; i < key.length; i++) {
-              replacementText += key[i];
+              if (key[i].indexOf('oppia-noninteractive') !== -1) {
```

Comment:
I'm actually a bit confused as to why you need the branching here. Doesn't the filter work either way?

Ditto below.

Reply:
It works. But there would be unnecessary calls to filter if we don't use the branching. Just to minimize the use of filter, I added this code.

Reply:
I would keep it simple. Otherwise you're doing yet another indexOf.

Reply:
That is something, I'm not sure about. Like which is worse for the performance: an indexOf or a call to filter! So, will change it to a simple call.